test/fixtures/filterings.yml

```diff
@@ -38,18 +38,6 @@ manager_3_3:
 manager_3_4:
   filter: manager_3
   permission: import_environments
-manager_4_0:
-  filter: manager_4
```

Comment:
looks like `manager_4` and `viewer_7` filters also need to be removed as they are no longer in use.